cmd/kubelet/app/server.go

```diff
@@ -330,6 +330,7 @@ func (s *KubeletServer) KubeletConfig() (*KubeletConfig, error) {
 		Runonce:                        s.RunOnce,
 		Port:                           s.Port,
 		ReadOnlyPort:                   s.ReadOnlyPort,
+		CadvisorProxyPort:              s.CadvisorPort,
```

Comment:
nit: We aren't doing this correctly in other places but it should be `CAdvisor` when used in Go code.